pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -254,13 +254,14 @@ async def request_stream(
         messages: list[ModelMessage],
         model_settings: ModelSettings | None,
         model_request_parameters: ModelRequestParameters,
+        run_context: RunContext[Any] | None = None,
```

Comment:
Please document the change to this signature as a breaking change for anyone who has implemented their own custom model. (And any other ways that we might have broken existing APIs that would only affect very deep usage..)